common/lib/xmodule/xmodule/course_module.py

```diff
@@ -1031,27 +1031,18 @@ class CourseFields(object):  # lint-amnesty, pylint: disable=missing-class-docst
     )
 
 
-class CourseModule(CourseFields, SequenceModule):  # pylint: disable=abstract-method
+class CourseBlock(
```

Comment:
@ormsbee @symbolist While we're renaming things... thoughts on calling this a `CourseRunBlock`, and/or renaming `SequenceModule` to `SubsectionBlock`?

It would take the class names out of sync with their block types (which'll always be 'course' and 'sequential'), but it would bring them in line with the prevailing user-facing terminology.

Reply:
Hmm, sounds like an interesting idea. 🤔 Will wait for @ormsbee's opinion.

Reply:
Sorry, I've been out for a while.

I'd rather hold off on the renaming for now. It doesn't cost any more to do the rename later if we want it, but there are these weird implications that I'd rather think about more. For instance:

1. Even if we change the root block to `CourseRunBlock`, we still have language everywhere in comments like, "reach up to the root course", and API calls like `get_course()`. Even though CourseRuns are the globally accepted DDD terminology for Open edX, we've already basically said we're going to accept that [Course and CourseRun are equivalent](https://github.com/edx/edx-platform/blob/master/docs/decisions/0001-courses-in-lms.rst) in the learning domain itself.
2. SubsectionBlock strongly implies hierarchy (it's in a Section), that I don't necessarily think is helpful or necessary. I could be convinced on this one, particularly if we have an alternative Sequence without all the baggage. But I'd rather keep it as is for now.

Reply:
Sounds good. 👍